.github/workflows/packages-upload-images.yml

```
@@ -130,6 +140,13 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.CI_WAZUH_AGENT_PACKAGES }}
 
+      - name: Request pkg_deb_agent_builder_ppc64le update
+        if: steps.changes.outputs.pkg_deb_agent_builder_ppc64le == 'true'
+        run: |
+          gh workflow run packages-upload-agent-images-ppc64le.yml --repo wazuh/wazuh-agent-packages -r ${{ github.ref_name }} -f docker_image_tag=${{ env.TAG }} -f system=deb -f architecture=ppc64le -f source_reference=${{ github.ref_name }}
```

Review the **yml** name: [packages-upload-agent-images-ppc.yml](https://github.com/wazuh/wazuh-agent-packages/pull/64/files#diff-150f4e669f4c57ec69baa1092150c68a2cee5c213a4fd3eecf1d2f6927fdf623R1)

```suggestion
          gh workflow run packages-upload-agent-images-ppc.yml --repo wazuh/wazuh-agent-packages -r ${{ github.ref_name }} -f docker_image_tag=${{ env.TAG }} -f system=deb -f architecture=ppc64le -f source_reference=${{ github.ref_name }}
```

---

done